documentation/config_file_puppetserver.markdown

```diff
@@ -26,6 +26,8 @@ The `puppetserver.conf` file contains settings for Puppet Server software. For a
 
     * `gem-home`: The location where JRuby looks for gems. It is also used by the `puppetserver gem` command line tool. If nothing is specified, JRuby uses the Puppet default `/opt/puppetlabs/server/data/puppetserver/jruby-gems`.
 
+    * `gem-path`: The complete "GEM_PATH" for jruby.  If set, it should include the `gem-home` directory, as well as any other directories that gems can be loaded from (including the vendored gems directory for gems that ship with puppetserver).  The default value is  `/opt/puppetlabs/server/data/puppetserver/jruby-gems:/opt/puppetlabs/server/data/puppetserver/vendored-jruby-gems`.
```

Comment:
Would it be better to say something like the following?

> The default value includes the value for the `gem-home` setting and `/opt/puppetlabs/server/data/puppetserver/vendored-jruby-gems`.  If the `gem-home` setting is not specified, the default value would be `/opt/puppetlabs/server/data/puppetserver/jruby-gems:/opt/puppetlabs/server/data/puppetserver/vendored-jruby-gems`.

Dunno, just wondered if it would be worth pointing out somehow that the first path derives from the `gem-home` setting.